electron-app/src/views/Statistics.vue

```diff
@@ -10,7 +10,9 @@
         No premium subscription detected. Statistics are only available to
         premium users. <br />
         To get a premium subscription please visit our
-        <a :href="$interop.premiumURL" target="_blank">website</a>.
+        <base-external-link href="https://rotki.com/products/">
```

Comment:
Same here no need to repeat the same URL everywhere

Reply:
See above.